qt/python/mantidqt/widgets/instrumentview/presenter.py

```diff
@@ -45,3 +49,59 @@ def rename_workspace(self, old_name, new_name):
 
     def show_view(self):
         self.container.show()
+
+    def get_current_tab(self):
+        return self.container.get_current_tab()
+
+    def get_render_tab(self):
+        return self.container.get_render_tab()
+
+    def get_pick_tab(self):
+        return self.container.get_pick_tab()
+
+    def select_render_tab(self):
+        self.container.select_tab(0)
+
+    def select_pick_tab(self):
+        self.container.select_tab(1)
+
+    def set_bin_range(self, min_x: float, max_x: float):
+        """Set the binning range on X-axis
+        """
+        self.container.set_range(min_x, max_x)
+
+    def close(self, workspace_name):
+        """
+        extend close()
+        :param workspace_name: str
+            workspace name
+        :return:
+        """
+        if InstrumentViewManager.last_view == self:
+            InstrumentViewManager.last_view = None
+        InstrumentViewManager.remove(self.ws_name)
+
+        super(InstrumentViewPresenter, self).close(self.ws_name)
+
+
+class InstrumentViewManager:
```

Comment:
Could we have some class/method documentation for these?

Reply:
@wdzhou I am not sure what this InstrumentViewManger is doing under the hood.  Could you add some documentation to clarify it?  Thanks